src/main/java/org/usfirst/frc4904/robot/subsystems/Elevator.java

```diff
@@ -0,0 +1,45 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+import org.usfirst.frc4904.standard.commands.motor.MotorIdle;
+import org.usfirst.frc4904.standard.custom.motioncontrollers.MotionController;
+import org.usfirst.frc4904.standard.custom.sensors.CANEncoder;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import org.usfirst.frc4904.standard.subsystems.motor.PositionSensorMotor;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class Elevator extends PositionSensorMotor {
+	public static final double UP_SPEED = 0.5;
+	public static final double DOWN_SPEED = -0.5;
+	public static final double ENCODER_TICKS = 1024; //todo: change to real thing
+	public static final double TICK_MULTIPLIER = 360/ENCODER_TICKS;
```

Comment:
The numerator here corresponds to distance per rotation, not degrees. You could even set that (the distance per rotation) to another variable to make it super clear.